job-dsls/jobs/srcclr_scan_job.groovy

```diff
@@ -0,0 +1,83 @@
+import org.kie.jenkins.jobdsl.Constants
+
+def repoFile = new URL(Constants.REPO_FILE_URL).text
+def repoList = repoFile.readLines()
```

Comment:
this could be `def repoList = new URL(Constants.REPO_FILE_URL).text.readLines()` since `repoFile` is not used anymore